src/test/java/com/wikia/webdriver/common/remote/discussions/UpvotePost.java

```diff
@@ -0,0 +1,35 @@
+package com.wikia.webdriver.common.remote.discussions;
+
+
+import com.google.common.collect.ImmutableMap;
+import com.jayway.jsonpath.JsonPath;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.remote.discussions.context.UpvoteContext;
+import com.wikia.webdriver.common.remote.operations.http.PostRemoteOperation;
+import com.wikia.webdriver.common.remote.operations.json.JsonToPostEntityMapper;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.PostEntity;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.elements.mercury.components.discussions.common.PostEntity'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)